src/components/editor/editor.js

```diff
@@ -387,15 +387,15 @@ export default class Editor extends React.Component {
     }, timeout);
   }
 
-  handleChangeRecipient(error) {
+  handleChangeRecipients(recipients, recipientsError) {
     this.setState(prevState => {
-      const errorState = {recipientsError: prevState.recipientsError, recipientsCcError: prevState.recipientsCcError, ...error};
-      return {...errorState, encryptDisabled: errorState.recipientsError || errorState.recipientsCcError || !prevState.recipients.length};
+      const errorState = {recipientsError, recipientsCcError: prevState.recipientsCcError};
```

Comment:
In the old logic we preserved the previous error state, the reason was that (I think) an error in the Cc field should not overwrite the error state of the To field.
It looks like with the new logic:
- prev To error state is always overwritten with new To error state
- Cc error state is never changed, always keeps prev Cc error state

Reply:
You are right. `handleChangeRecipients` now doesn't know if it was a `To` or `CC` change. Let me adjust it.